apps/neuvector/neuvector/neuvector.yaml

```diff
@@ -77,34 +70,6 @@ spec:
           cpu: 1
           memory: 5Gi
       controller:
-        ingress:
-          traefik.ingress.kubernetes.io/router.tls: "true"
-        tolerations:
-          - key: "CriticalAddonsOnly"
-            operator: "Equal"
-            value: "true"
-            effect: "NoSchedule"
-        affinity:
-          podAntiAffinity:
-            preferredDuringSchedulingIgnoredDuringExecution:
-              - weight: 100
-                podAffinityTerm:
-                  labelSelector:
-                    matchExpressions:
-                      - key: app
-                        operator: In
-                        values:
-                          - neuvector-controller-pod
-                  topologyKey: "kubernetes.io/hostname"
-          nodeAffinity:
-            preferredDuringSchedulingIgnoredDuringExecution:
-              - weight: 1
-                preference:
-                  matchExpressions:
-                    - key: kubernetes.azure.com/mode
-                      operator: In
-                      values:
-                        - system
```

Comment:
what's with all these values getting removed throughout the file?

Reply:
Was getting errors around the syntax in the file which was still following a fluxv1 config as far as I could tell so I checked out the config in cft and used that as a guide. I can see from the neuvector-helm repo that the tolerations and affinity values are valid so I can add these back in if you think they need to be there for sds.

Reply:
yes they need to be there for SDS